app/presenters/finder_presenter.rb

```diff
@@ -131,6 +133,33 @@ def from
     organisations + people
   end
 
+  def other
+    if applies_to
+      {
+        "Applies to" => applies_to.html_safe,
+      }
+    end
+  end
+
+  def applies_to
```

Comment:
It feels like this method would be better as two separate methods: one boolean method that determines if the document applies to all nations, and another to construct the meta data fragment. Then the boolean method could be used in the `other` method above. Something like this:

```ruby

def other
  unless applies_to_all_nations?
    { "Applies to" => applicable_nations_html_fragment }
  end
end

def applies_to_all_nations?
  content_item.details.nation_applicability && content_item.details.nation_applicability.any?
end

def applicable_nations_html_fragment
  applies_to = content_item.details.nation_applicability.map(&:titlecase)
  alternative_policies = applicability.alternative_policies.map do |alternative|
    if alternative.alt_policy_url
      link_to(alternative.nation.titlecase, n.alt_policy_url, ({rel: 'external'} if is_external?(n.alt_policy_url)))
    else
      alternative
    end
  end
  "#{applies_to.to_sentence} (see policy for #{alt_policies.to_sentence})".html_safe
end
```

Also, is that if/else is required? Looking at the [policy-publisher code](https://github.com/alphagov/policy-publisher/pull/34/files#diff-00c608fb3b501b4f92d04bcf03760afeR6), it isn't possible to set a nation as being inapplicable without supplying an alternative policy URL. And if it was, would the array contain strings and not the hash that it contains when a URL is provided?